src/app/components/Pagination/index.tsx

```diff
@@ -55,7 +55,7 @@ const visibilityToMediaQuery = (visibility: string) =>
     [VISIBILITY.TABLET_UP]: `display: none; @media (min-width: ${GROUP_3_MIN_WIDTH_BP}) {
       display: inline-block;
     }`,
-    [VISIBILITY.DESKTOP_ONLY]: `display: none; @media (min-width: ${GROUP_4_MIN_WIDTH_BP}) {
+    [VISIBILITY.DESKTOP_ONLY]: `display: none; @media (min-width: 1008px) {
```

Comment:
As mentioned in my comment on Slack, I think all of these queries might need to be updated. The individual values of `GROUP_X_MIN/MAX_WIDTH` must not be used on their own, as they are just a number - they need to have `rem` added as a suffix (see https://github.com/bbc/simorgh/blob/125718d653b00808814cf1563d7919e699a7c881/src/app/components/ThemeProvider/mediaQueries.ts#L26-L42)

What if we were to try the following:

```suggestion
    [VISIBILITY.DESKTOP_ONLY]: `display: none; ${GROUP_4_MIN_WIDTH} {
```

If that works, could we also replace the other visibility queries here to use the shortcut media query, so we don't need to hand crank the min/max width?

Reply:
grand - good sleuthing, KT